src/engine/profiler.js

```diff
@@ -106,6 +106,12 @@ class ProfilerFrame {
          * @type {number}
          */
         this.depth = depth;
+
+        /**
+         * A summarized count of the number of calls to this frame.
+         * @type {number}
+         */
+        this.count = 1;
```

Comment:
Hmmm why isn't this initialized to `0`? Increments creates a new ProfilerFrame if one doesn't exist, and then increments this count anyways, so that means a newly created frame would get incremented to `2` which seems weird...

Reply:
Hmm. I faintly remember this being 1 due to the branches creating frames not incrementing them. In that case this was an oversight after refactoring.